tests/converter/test_text.py

```diff
@@ -10,9 +10,22 @@
     "fontSize": 12,
     "textAlignVertical": "CENTER",
     "textAlignHorizontal": "CENTER",
-    "fillPaints": [{"type": "SOLID", "color": FIG_COLOR[0]}],
+    "fillPaints": [{"type": "SOLID", "color": FIG_COLOR[0], "opacity": 1, "visible": True}],
```

Comment:
When these 2 properties are missing, the converter fails: they're set as optional but they're retrieved with `fig_text["opacity"]` instead of a `fig_text.get("opacity"). 

Should I change the converter to use the safer access method or we can trust they'll be always present and adding them to the test is enough?

Reply:
Ah yes. So far, all documents had it, so I think fixing the test is enough (I already did some of this today). :+1: